Makefile

```diff
@@ -49,3 +49,7 @@ clean:
 	$(RM) *.a *.~ *.exe *.o
 	$(MAKE) -C libmincrypt clean
 
+install:
+	install -m 755 unpackbootimg $(PREFIX/bin)
```

Comment:
This results in:
```
$ make install
install -m 755 unpackbootimg
install: missing destination file operand after 'unpackbootimg'
Try 'install --help' for more information.
make: *** [Makefile:53: install] Error 1
```

So I think you meant `$(PREFIX)/bin` instead for this and the next line?

Reply:
Should probably be unpackbootimg$(EXE) and mkbootimg$(EXE) technically, as well, in keeping with the rest of the Makefile.

Reply:
Using == just compares whether both pointers are in same location. Hence strcmp is used to compare if both string arw same slight mistake add == 0. Change made in file

Reply:
Not sure why you used curly brackets instead of round where the rest of the file uses round. Again, stylistic consistency is important when contributing to a project.